src/orion/algo/space/configspace.py

```diff
@@ -71,12 +71,19 @@ def dimension(self, dim: Dimension) -> None:
     def real(self, dim: Real) -> FloatHyperparameter:
         """Convert a real dimension into a configspace equivalent"""
         if dim.prior_name in ("reciprocal", "uniform"):
-            a, b = dim._args
+
+            # NB: scipy uniform [loc, scale], configspace [min, max] with max = loc + scale, loc = min
+            if dim.prior_name == "uniform":
+                loc, scale = dim._args
```

Comment:
Could they be in `kwargs` as well?

Reply:
They could if the space was build by hand
that is going to add a lot more ifs though

Reply:
I found a way to make it work